src/OSPSuite.Core/Domain/Services/ISimModelManager.cs

```diff
@@ -39,5 +37,7 @@ public interface ISimModelManager
       ///    Event raised when simulation is terminated (either after normal termination or cancel)
       /// </summary>
       event EventHandler Terminated;
+
+      Task<SimulationRunResults> RunSimulationAsync(IModelCoreSimulation simulation, CancellationToken cancellationToken = default, SimulationRunOptions simulationRunOptions = null);
```

Comment:
Can you add a comment to the function explaining usage of the cancellation token and what it means if the value is undefined